Dockerfile

```
@@ -1,5 +1,13 @@
 FROM python:3
-COPY . /workdir/
 WORKDIR /workdir
-RUN pip install --requirement requirements.txt
+COPY . .
+RUN pip install \
+    black \
+    codecov \
```

@evaristor y @xwalls para quitar un poco de los errores del actions y avanzar a otros podemos instalar esta extensión:
```suggestion
    codecov \
    data-science-types \
```